azure_monitor/tests/metrics/test_metrics.py

```diff
@@ -294,7 +298,7 @@ def test_measure_to_envelope(self, logger_mock):
         self.assertIsInstance(envelope.data.base_data.metrics[0], DataPoint)
         self.assertEqual(envelope.data.base_data.metrics[0].ns, "testdesc")
         self.assertEqual(envelope.data.base_data.metrics[0].name, "testname")
-        self.assertEqual(envelope.data.base_data.metrics[0].value, 0)
+        self.assertEqual(envelope.data.base_data.metrics[0].value, 1)
```

Comment:
Is the default value 1 now, or why this is different now?

Reply:
App insights does not have a way to process MinMaxSumCount aggregator so I am just using count value as a placeholder for now. Before it was 0 because we ignored that metric type when exporting.